src/QuadGK.jl

```diff
@@ -333,11 +337,17 @@ function gauss{T<:AbstractFloat}(::Type{T}, N::Integer)
     return (x, w)
 end
 
-# Compute 2n+1 Kronrod points x and weights w based on the description in
-# Laurie (1997), appendix A, simplified for a=0, for integrating on [-1,1].
-# Since the rule is symmetric only returns the n+1 points with x <= 0.
-# Also computes the embedded n-point Gauss quadrature weights gw (again
-# for x <= 0), corresponding to the points x[2:2:end].  Returns (x,w,wg).
+gauss(N::Integer) = gauss(Float64, N)
+
+"""
+    kronrod([T,] n)
+
+Compute `2n+1` Kronrod points `x` and weights `w` based on the description in
+Laurie (1997), appendix A, simplified for `a=0`, for integrating on `[-1,1]`.
+Since the rule is symmetric, this only returns the `n+1` points with `x <= 0`.
+The function Also computes the embedded `n`-point Gauss quadrature weights `gw`
+(again for `x <= 0`), corresponding to the points `x[2:2:end]`. Returns `(x,w,wg)`.
+"""
```

Comment:
```
Given these points and weights, the estimated integral `I` and error `E` can be computed for an
integrand `f(x)` as follows:

    x,w,wg = kronrod(n)
    fx⁰ = f(x[end])      # f(0)
    x⁻ = x[1:end-1]     # the x < 0 Kronrod points
    fx = f.(x⁻) .+ f.((-).(x⁻))    # f(x < 0) + f(x > 0)
    I = sum(fx .* w[1:end-1]) + fx⁰ * w[end]
    if isodd(n)
        E = abs(sum(fx[2:2:end] .* wg[1:end-1]) + fx⁰*wg[end] - I)
    else
        E = abs(sum(fx[2:2:end] .* wg[1:end])- I)
    end
```

Also, should have a similar comment to above explaining `T` and noting that it is O(N^2).